### DISCUSSION TITLE
建议加一个修改翻译链接的功能(It is recommended to add a function to modify the Google Translate link)

### DISCUSSION THREAD

**Discussion by Alan3344** [YiiGuxing/TranslationPlugin#1417] [Ideas]
首先你做的插件对于写代码的我来说非常好用!
谷歌翻译是不错的,我平常用的也是最多的,因为他国际化没有广告,但是在中国大陆地区由于防火墙的存在, https://translate.google.com/ 无法正常访问,如果不通过VPN的话.
但是 https://translate.google.cn/ 是可以访问的希望加一个输入框以修改域名或者之类的功能,有道和百度等需要AppKey不方便.
希望作者可以加以修改,谢谢!
By English:
First of all, the plugin you made is very useful for me who writes code!
Google Translate is good, and I usually use it the most, because it is internationalized without advertisements, but in mainland China, due to the existence of the firewall, https://translate.google.com/ cannot be accessed normally without VPN. .
However, https://translate.google.cn/ is accessible. I hope to add an input box to modify the domain name or other functions. It is inconvenient for Youdao and Baidu to require AppKey.
I hope the author can modify it, thank you!

**Comment by YiiGuxing** [comment]
`translate.google.cn` 的接口早已无法使用。
